unit-tests/func/MUR/l500/test-turn-off-IR-Reflectivity.cpp

```diff
@@ -0,0 +1,53 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
caps in filename
I think this test is more about reflectivity than MUR, but up to you

Reply:
(inside auto-turn-off?)

Reply:
(or in force-mur, even)

Reply:
Merged with "test-auto-set-options"